### ISSUE TITLE
Fails with MethodError: no method matching decode_effects(::UInt8)

### ISSUE THREAD

**Issue by cafaxo** [JuliaDebug/Cthulhu.jl#287]
On latest Julia and Cthulhu master I get
```
ERROR: MethodError: no method matching decode_effects(::UInt8)
Closest candidates are:
  decode_effects(::UInt32) at compiler/types.jl:137
Stacktrace:
 [1] get_effects(codeinst::Core.CodeInfo)
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:241
 [2] _descend(term::REPL.Terminals.TTYTerminal, interp::Cthulhu.CthulhuInterpreter, mi::Core.MethodInstance; override::Core.Compiler.InferenceResult, debuginfo::Cthulhu.DInfo.DebugInfo, optimize::Bool, interruptexc::Bool, iswarn::Bool, hide_type_stable::Bool, verbose::Nothing, remarks::Bool, with_effects::Bool, inline_cost::Bool, type_annotations::Bool)
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:345
 [3] _descend(term::REPL.Terminals.TTYTerminal, interp::Cthulhu.CthulhuInterpreter, mi::Core.MethodInstance; override::Nothing, debuginfo::Cthulhu.DInfo.DebugInfo, optimize::Bool, interruptexc::Bool, iswarn::Bool, hide_type_stable::Nothing, verbose::Nothing, remarks::Bool, with_effects::Bool, inline_cost::Bool, type_annotations::Bool)
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:474
 [4] _descend(term::REPL.Terminals.TTYTerminal, args::Any; interp::Core.Compiler.NativeInterpreter, kwargs::Base.Pairs{Symbol, Bool, Tuple{Symbol}, NamedTuple{(:iswarn,), Tuple{Bool}}})
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:595
 [5] __descend_with_error_handling(args::Any; terminal::Any, kwargs::Base.Pairs{Symbol, V, Tuple{Vararg{Symbol, N}}, NamedTuple{names, T}} where {V, N, names, T<:Tuple{Vararg{Any, N}}})
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:188
 [6] _descend_with_error_handling(f::Any, argtypes::Any; kwargs::Base.Pairs{Symbol, Bool, Tuple{Symbol}, NamedTuple{(:iswarn,), Tuple{Bool}}})
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:179
 [7] descend_code_typed(::Any, ::Vararg{Any}; kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:136
 [8] descend_code_typed(::Any, ::Any)
   @ Cthulhu ~/.julia/packages/Cthulhu/2PHyP/src/Cthulhu.jl:136
 [9] top-level scope
   @ REPL[3]:1
```
Working on a MWE...

**Comment by ianatol** [comment]
I can't say for sure without an MWE, but this should be fixed by #281

**Comment by cafaxo** [comment]
Yes, I cannot reproduce it anymore.
